### PR TITLE
feat: Parse ms:laurl and mspr:pro

### PR DESCRIPTION
**Pull request by forbesjo**
This PR is to address https://github.com/google/shaka-player/issues/484
See http://download.microsoft.com/download/2/3/8/238F67D9-1B8B-48D3-AB83-9C00112268B2/PlayReady%20Header%20Object%202015-08-13-FINAL-CL.PDF for `mspr:pro`

Todo:
- [x] Add tests
- [x] Dash.js attribution
- [x] More comments to  `getPlayReadyLicenseServerURL_`

### CODE REVIEW

File: lib/dash/content_protection.js [hunk 1]
```diff
@@ -276,7 +357,19 @@ shaka.dash.ContentProtection.convertElements_ = function(
           goog.asserts.assert(!element.init || element.init.length,
                               'Init data must be null or non-empty.');
           var initData = element.init || defaultInit;
-          return [ManifestParserUtils.createDrmInfo(keySystem, initData)];
+          var drmInfo = ManifestParserUtils.createDrmInfo(keySystem, initData);
+
+          // extract Widevine license URL
+          if (keySystem === 'com.widevine.alpha') {
```
Comment (mjneil) [inline lib/dash/content_protection.js:455]: Is this key system only used for the microsoft specific widevine?

Comment (forbesjo) [reply lib/dash/content_protection.js:455]: This will appear for the Widevine pssh node as well

Comment (mjneil) [reply lib/dash/content_protection.js:455]: is having the empty string for the `licenseServerUri` what is desired in the pssh node case?

Comment (forbesjo) [reply lib/dash/content_protection.js:455]: Yes

**Comment by mjneil** [comment]
lgtm

**Comment by mjneil** [comment]
Why no longer check the `LUI_URL` in the latest commit?

**Comment by forbesjo** [comment]
According to the [PlayReady Object document](http://download.microsoft.com/download/2/3/8/238F67D9-1B8B-48D3-AB83-9C00112268B2/PlayReady%20Header%20Object%202015-08-13-FINAL-CL.PDF) `LUI_URL` "contains the URL for a non-silent license acquisition web page." Meaning that it can be used as an intermediary step requiring user interaction. I dropped it since the sources that we're trying to support with this PR don't have this tag and it seems like this would require more logic to actually work properly.

**Review by mjneil — APPROVED** [review]

**Review by mjneil — APPROVED** [review]
tested and lgtm

**Pull request merged**